app/src/main/java/org/mozilla/fenix/library/bookmarks/edit/EditBookmarkFragment.kt

```diff
@@ -166,7 +166,7 @@ class EditBookmarkFragment : Fragment(R.layout.fragment_edit_bookmark) {
                             bookmarkNode?.let { bookmark ->
                                 FenixSnackbar.make(
                                     view = activity.getRootView()!!,
-                                    isDisplayedWithBrowserToolbar = true
+                                    isDisplayedWithBrowserToolbar = false
```

Comment:
We added this because if you add a bookmark from the browser and then click the "edit" button in the snackbar, you get to this view, and then if you save the changes, you get back to the browser. Without adding a flag this will be wrong somewhere :/

Reply:
I added a safe argument flag to this fragment so we know if the snackbar should account for the toolbar being visible or not.
It's default value is set to false, so we only need to add it when we navigate to the EditBookmarkFragment from the "edit" button of the snackbar or from the "star" icon that navigates to the edit fragment if the page is already added to the bookmarks.